notebooks/02_model/sar_single_node_deep_dive.ipynb

```diff
@@ -87,7 +87,7 @@
   },
```

Comment:
<p>same as before, not sure if we are using the correct test set</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
curious, let me look into it